docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,259 @@
+* **Python:** To follow along, you need Python 3.9 or 3.10.
+
+## Installing the Flink backend for Ibis
+
+We use a Python client to explore data in Kafka topics. You can install it, alongside the Flink backend for Ibis, with `pip`, `conda`, `mamba`, or `pixi`:
+
+::: {.panel-tabset}
+
+## Using `pip`
+
+```bash
+pip install ibis-framework apache-flink kafka-python
+```
+
+## Using `conda`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+## Using `mamba`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+## Using `pixi`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+:::
+
+## Spinning up the services using Docker Compose
+
+The [claypotai/ibis-flink-example GitHub repository](https://github.com/claypotai/ibis-flink-example) includes the relevant Docker Compose configuration for this tutorial. Clone the repository, and run `docker compose up` from the cloned directory to create Kafka topics, generate sample data, and launch a Flink cluster.
+
+```bash
+git clone https://github.com/claypotai/ibis-flink-example.git
+cd ibis-flink-example
+docker compose up
+```
+
+::: {.callout-tip}
+If you don't intend to try [remote execution](#remote-execution), you can start only the Kafka-related services with `docker compose up kafka init-kafka data-generator`.
+:::
+
+After a few seconds, you should see messages indicating your Kafka environment is ready:
+
+```bash
+ibis-flink-example-init-kafka-1      | Successfully created the following topics:
+ibis-flink-example-init-kafka-1      | payment_msg
+ibis-flink-example-init-kafka-1      | sink
+ibis-flink-example-init-kafka-1 exited with code 0
+ibis-flink-example-data-generator-1  | Connected to Kafka
+ibis-flink-example-data-generator-1  | Producing 20000 records to Kafka topic payment_msg
+```
+
+This example uses mock payments data. The `payment_msg` Kafka topic contains messages in the following format:
+
+```json
+{
+    "createTime": "2023-09-20 22:19:02.224",
+    "orderId": 1695248388,
+    "payAmount": 88694.71922270155,
+    "payPlatform": 0,
+    "provinceId": 6
+}
+```
+
+In a separate terminal, we can explore what these messages look like:
+
+```{python}
+from itertools import islice
+
+from kafka import KafkaConsumer
+
+consumer = KafkaConsumer("payment_msg")
+for msg in islice(consumer, 3):
+    print(msg)
+```
+
+## Running the tutorial
+
+This tutorial uses the Flink backend for Ibis to process the aforementioned payment messages. You can choose to either [run it locally](#local-execution) or [submit a job to an already-running Flink cluster](#remote-execution).
```

Comment:
suggestion: "uses Ibis with the Flink backend"

Reply:
Will do!